### ISSUE TITLE
error when user forgot `on User` on fragment

### ISSUE THREAD

**Issue by sibelius** [relayjs/eslint-plugin-relay#39]
I've got the error below when I forgot to have `on User` on my fragment for instance:

```jsx
user: graphql`
    fragment MyComponent_user {
}
```

```jsx
TypeError: Cannot read property 'kind' of null
    at walkAST (/Users/sibelius/Dev/app-test/node_modules/eslint-plugin-relay/src/rule-unused-fields.js:21:14)
    at getGraphQLFieldNames (/Users/sibelius/Dev/app-test/node_modules/eslint-plugin-relay/src/rule-unused-fields.js:51:3)
    at Program:exit.templateLiterals.forEach.templateLiteral (/Users/sibelius/Dev/app-test/node_modules/eslint-plugin-relay/src/rule-unused-fields.js:132:31)
    at Array.forEach (<anonymous>)
    at Program:exit (/Users/sibelius/Dev/app-test/node_modules/eslint-plugin-relay/src/rule-unused-fields.js:129:24)
```

Should we handle this case?

**Comment by kassens** [comment]
Yes, that's a syntax error and the `unused-fields` rule should ignore that tag. The `graphql-syntax` rule should handle the error instead.

**Comment by kassens** [comment]
Just released 0.0.27 with the fix, let me know if that doesn't fix it!
